### PR TITLE
Nc news 4

### PR DESCRIPTION
**Pull request by JamieLewis748**

### CODE REVIEW

File: __tests__/integration.tests.js [hunk 1]
```diff
@@ -67,3 +67,38 @@ describe('ALL /notapath', () => {
...
+describe("/api/articles/:article_id", () => {
+    test('GET: 200, should respond with an array containing a single object with given id and correct keys', () => {
+        return request(app)
+            .get("/api/articles/2")
+            .expect(200)
+            .then((response) => {
+                const article = response.body;
+                expect(article).toHaveProperty("author");
+                expect(article).toHaveProperty("title");
+                expect(article).toHaveProperty("article_id");
```
Comment (B7Bprog) [inline __tests__/integration.tests.js]: Please check the actual article id here, to make sure it's 2 as well, so we've received the right one.

File: controllers/articles.controller.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const { selectArticleById } = require("../models/articles.model");
+
+exports.getArticleById = (req, res, next) => {
+    const articleId = req.params.article_id;
+    selectArticleById(articleId).then((article) => {
+        res.status(200).send(article);
```
Comment (B7Bprog) [inline controllers/articles.controller.js]: Please send the article back in a new object on a key of 'article'.

**Review by B7Bprog — CHANGES_REQUESTED** [review]
Hi Jamie, please see my comments and refactor your code, then once all tests are passing you can merge it.

File: __tests__/integration.tests.js (hunk 1 above)
Comment (JamieLewis748) [reply __tests__/integration.tests.js]: expect(article).toHaveProperty("article_id", 2);

**Pull request merged**